examples/convert.py

```diff
@@ -0,0 +1,53 @@
+import tensorflow as tf
+from tensorflow.python.platform import gfile
+import tensorflow.keras as K
+
+from tensorflow.python.framework import graph_util
+from tensorflow.python.framework import graph_io
+from tensorflow_encrypted.convert import convert
+import numpy as np
+
+import os
+
+
+def export_vgg16():
+    vgg16 = tf.keras.applications.VGG16()
+
+    sess = K.backend.get_session()
+
+    pred = [None]
+    pred_node_names = [""]
+    pred_node_names[0] = "output"
+    pred[0] = tf.identity(vgg16.outputs[0], name=pred_node_names[0])
```

Comment:
Why do you define `pred = [None]` and then reassign it directly `pred[0] = ...`?
Same question for `pred_node_names `

Reply:
Not reason really. This code was borrowed from elsewhere and I haven't make it nice nice yet.